protocol/x/clob/keeper/match_state.go

```diff
@@ -24,17 +59,26 @@ func (k Keeper) GetLastTradePriceForPerpetual(
 	return types.Subticks(result.Value), true
 }
 
-// SetLastTradePriceForPerpetual sets the last trade price for a perpetual.
-func (k Keeper) SetLastTradePriceForPerpetual(
+// SetTradePricesForPerpetual sets the trade prices for a perpetual.
```

Comment:
Can we add some color in the docstring for
- what is a trade price?
- sets in transient state store
- where trade price is used
- where this is supposed to be called in?

Reply:
maybe `SetLastTradePriceForPerpetual` too